kuma/static/styles/base/icons.scss

```diff
@@ -265,6 +267,15 @@ button[type='submit'] .icon-check-mark {
     vertical-align: text-bottom;
 }
 
+/* TODO: Need to remove the need for this be removing the
+   styling added from the React side */
```

Comment:
I don't understand the comment.

Reply:
There is styling on the React side that needs to be moved here. It just makes more sense to have all styling related to icons in one place. But, seeing that the history button is being removed, this is moot now ;)